modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryRelatedObjectsResourceImpl.java

```diff
@@ -144,6 +203,21 @@ private DefaultDTOConverterContext _getDTOConverterContext(
 			contextUser);
 	}
 
+	private ObjectDefinition _getRelatedObjectDefinition(
```

Comment:
We could move this to a new method in the local service and use it all the times we implement this private method